src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/Http3ConnectionTests.cs

```diff
@@ -262,7 +262,6 @@ public async Task SETTINGS_MaxFieldSectionSizeSent_ServerReceivesValue()
         }
 
         [Fact]
-        [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/34685")]
```

Comment:
There are several more tests in this file linking to the same issue. Are they all ready to unquarantined?

Reply:
Will check

Reply:
You were right. They were not mentioned in the issue and I failed to notice them. Will check the other files too then.